portfolio/src/main/webapp/style.css

```diff
@@ -1,13 +1,74 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+* {
+    box-sizing: border-box;
+}
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  padding: 10px;
+  background: #e6e6e6;
 }
 
-#greeting-container {
-  margin-top: 20px;
+.header-home {
+  padding: 30px;
+  text-align: center;
+  background-image: linear-gradient(rgba(0, 0, 0, 0.2),rgba(0, 0, 0, 0.2)), 
+        url('/images/floridalake.jpg');
+  background-size: 100%, 100%
+}
+
+.header-home h1 {
+  font-size: 50px;
+  color: white;
+}
+
+.topnav {
+  overflow: hidden;
+  background-color: #1c1c1c;
+}
+
+.topnav a {
+  float: left;
+  display: block;
+  color: #f2f2f2;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+}
+
+.topnav a:hover {
+  background-color: #e6e6e6;
+  color: black;
 }
 
-#quote-container {
+.content-row{
+    background-color: white; 
+    display: table;
+    padding: 20px;
+    margin-top: 20px;
+}
+
+.leftcolumn {   
+  float: left;
+  width: 65%;
+}
+
+.rightcolumn {
+  float: left;
+  width: 35%;
+  padding-left: 20px;
+}
+
+.footer {
+  padding: 20px;
+  text-align: center;
+  background: #686868;
```

Comment:
The footer is a little dark, especially since the hyperlinks are purple and blue, which could be a struggle for someone with red/blue colorblindness. Consider a lighter gray such as #dbdbdb or even a green or blue pulled from your header picture.